packages/orbit-tailwind-preset/src/presets/__tests__/__snapshots__/configs.test.ts.snap

```diff
@@ -2517,14 +2477,17 @@ exports[`orbitPreset should match snapshot 1`] = `
     },
     "padding": {
       "0": "0px",
+      "button-large": "52px",
```

Comment:
I think instead of introducing those new tokens you can use `formBox` tokens, the values are the same 

If you take a look at the `formBox` json definition, it sets height tokens for buttons as deprecated tokens :)